### PR TITLE
Missing tests for float comparison methods

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Updated comparison methods so that each of them calls basic `equals`.
Added tests.
@marcusnaslund

### CODE REVIEW

File: test/sdk/MathTest.ooc [hunk 1]
```diff
@@ -55,47 +55,75 @@ MathTest: class extends Fixture {
...
 			expect(1.99999 equals(2.0), is true)
+
+			nearZero := (0.1 + 0.1 + 0.1) - 0.3
+			expect(nearZero equals(0.0), is true)
+			expect(nearZero lessOrEqual(0.0), is true)
+			expect(nearZero greaterOrEqual(0.0), is true)
+			expect(nearZero greaterThan(0.0), is false)
+			expect(nearZero lessThan(0.0), is false)
+
+			nearZero = 0.3 - (0.1 + 0.1 + 0.1)
```
Comment (sebastianbaginski) [inline test/sdk/MathTest.ooc]: I wanted to test both negative and positive epsilon, that's why this variable is reassigned here.

File: test/sdk/MathTest.ooc [hunk 2]
```diff
@@ -55,47 +55,75 @@ MathTest: class extends Fixture {
...
+
 			expect(Float maximum(-1.2f, -1.1f), is equal to(-1.1f) within(floatTolerance))
 			expect(Float maximum(-1.2f, 0.f), is equal to(0.f) within(floatTolerance))
 			expect(Float minimum(-1.2f, -1.1f), is equal to(-1.2f) within(floatTolerance))
 			expect(Float minimum(-1.2f, 0.f), is equal to(-1.2f) within(floatTolerance))
-			
+
 			expect(10.f squared(), is equal to(100.f) within(floatTolerance))
-			
+
```
Comment (marcusnaslund) [inline test/sdk/MathTest.ooc]: Aaaaaaaaah @thomasfanell

Comment (sebastianbaginski) [inline test/sdk/MathTest.ooc]: I think the tab should not be there or ?

File: test/sdk/MathTest.ooc [hunk 3]
```diff
@@ -55,47 +55,75 @@ MathTest: class extends Fixture {
...
 			expect(Float linearInterpolation(2.0f, 5.0f, 0.5f), is equal to(3.5f) within(floatTolerance))
 			expect(Float linearInterpolation(-9.0f, 1.0f, 0.1f), is equal to(-8.f) within(floatTolerance))
+
+			nearZero := (0.1f + 0.1f + 0.1f) - 0.3f
+			expect(nearZero equals(0.0), is true)
```
Comment (marcusnaslund) [inline test/sdk/MathTest.ooc]: `0.0f`?

File: test/sdk/MathTest.ooc (hunk 2 above)
Comment (marcusnaslund) [inline test/sdk/MathTest.ooc]: That's right. I blame VS Code.

File: test/sdk/MathTest.ooc (hunk 3 above)
Comment (sebastianbaginski) [inline test/sdk/MathTest.ooc]: updated

**Pull request merged**